salesagility/SuiteCRM-Core
# Fix #458 - Fix hard coded labels

## Description




Fix #458 
## Motivation and Context


## How To Test This

1. Install SuiteCRM
2. Go to Module Loader within Studio
3. Add a language pack
4. Set language in Locale
5. See all labels updated correctly


## Types of changes


### Final checklist
